jerry-core/parser/js/js-lexer.c

```diff
@@ -1937,6 +1962,13 @@ lexer_construct_number_object (parser_context_t *context_p, /**< context */
     const uint8_t *src_p = context_p->token.lit_location.char_p;
     const uint8_t *src_end_p = src_p + length - 1;
 
+#if ENABLED (JERRY_ES2015)
+    if (src_p[1] == LIT_CHAR_LOWERCASE_O)
```

Comment:
This should be `src_p[1] ==LEXER_TO_ASCII_LOWERCASE (LIT_CHAR_LOWERCASE_O)`

Reply:
Good catch! You are right :)